# Request
string mapfile

---

# Response
bool success
string message
